src/shims/fs.rs

```diff
@@ -427,17 +427,23 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
             this.read_scalar(dirfd_op)?.to_machine_isize(&*this.tcx)?.try_into().map_err(|e| {
                 err_unsup_format!("Failed to convert pointer sized operand to integer: {}", e)
             })?;
-        // we only support interpreting `path` as an absolute directory or as a directory relative
-        // to `dirfd` when the latter is `AT_FDCWD`. The behavior of `statx` with a relative path
-        // and a directory file descriptor other than `AT_FDCWD` is specified but it cannot be
-        // tested from `libstd`. If you found this error, please open an issue reporting it.
+        // We only support:
+        // * interpreting `path` as an absolute directory,
+        // * interpreting `path` as a path relative to `dirfd` when the latter is `AT_FDCWD`, or
+        // * interpreting `dirfd` as any file descriptor when `path` is empty and AT_EMPTY_PATH is
+        // set.
+        // The behavior of `statx` with a relative path and a directory file descriptor other than
+        // `AT_FDCWD` is specified but it cannot be tested from `libstd`. If you found this error,
```

Comment:
I'd change this to: "Other behaviors cannot be tested from `libstd" and thus are not implemented. If you found this error, please open an issue reporting it.